### PR TITLE
fix: make quick start clear and concise

### PR DESCRIPTION
**Pull request by lowzj**
Signed-off-by: lowzj <zj3142063@gmail.com>

### CODE REVIEW

File: docs/zh-cn/userguide/multi_machines_deployment.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    * **dfclient1** 用于部署 Dragonfly 客户端
+2. 以上每台机器上都装有 docker 容器引擎。
+
+## 步骤 1：部署服务端（Supernode） 
+
+在机器 `dfsupernode` 上部署 Dragonfly 服务端（Supernode）：
+
+```bash
+docker run -d --name supernode --restart=always -p 8001:8001 -p 8002:8002 \
+    dragonflyoss/supernode:0.3.0 -Dsupernode.advertiseIp=dfsupernode
```
Comment (Starnop) [inline docs/zh-cn/userguide/multi_machines_deployment.md]: s/dfsupernode/$DFSUPERNODE_IP. WDYT?

File: docs/zh-cn/userguide/multi_machines_deployment.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+**启动客户端**
+
+```bash
+docker run -d --name dfclient --restart=always -p 65001:65001 \
+    -v /etc/dragonfly:/etc/dragonfly \
+    dragonflyoss/dfclient:v0.3.0 --registry https://index.docker.io
+```
+
+> **提示**：`--registry`参数指定镜像仓库地址，这里为官方镜像仓库，您也可以设置为其他仓库地址。
```
Comment (Starnop) [inline docs/zh-cn/userguide/multi_machines_deployment.md]: /这里/ https://index.docker.io？

File: docs/zh-cn/quickstart.md [hunk 3]
```diff
@@ -1,97 +1,65 @@
...
-```bash
-docker pull dragonflyoss/supernode:0.3.0
-```
+本文档所有操作步骤均使用 docker 容器在同一台机器上完成，所以请确保您的机器上已经安装并启动 docker 容器引擎。您也可以参考文档：[多机部署](userguide/multi_machines_deployment.md) 来体验 Dragonfly。
 
-2. 启动 SuperNode
+## 步骤 1：部署 Dragonfly 服务端（Supernode）
 
 ```bash
 docker run -d -p 8001:8001 -p 8002:8002 dragonflyoss/supernode:0.3.0 -Dsupernode.advertiseIp=127.0.0.1
```
Comment (Starnop) [inline docs/zh-cn/quickstart.md]: --name supernode --restart=always

File: docs/zh-cn/userguide/multi_machines_deployment.md (hunk 1 above)
Comment (lowzj) [reply docs/zh-cn/userguide/multi_machines_deployment.md]: `dfsupernode` is declared above, it could be ip, hostname, domain, or shell variable.

**Comment by Starnop** [comment]
LGTM.

**Pull request merged**